inference-engine/tests/functional/plugin/shared/include/behavior/core_integration.hpp

```diff
@@ -431,8 +431,7 @@ TEST_P(IEClassBasicTestP, ImportNetworkThrows) {
     SKIP_IF_CURRENT_TEST_IS_DISABLED()
     Core ie = createCoreWithTemplate();
 
-    if (deviceName == CommonTestUtils::DEVICE_CPU ||
-        deviceName == CommonTestUtils::DEVICE_GPU) {
+    if (deviceName == CommonTestUtils::DEVICE_GPU) {
```

Comment:
Why did you remove GPU?

Reply:
I excluded the CPU from the condition because now the CPU plugin doesn't throw the required exception from ImportNetwork.